### PR TITLE
Increase logo spin. Change title of main page.

### PR DESCRIPTION
**Pull request by vcjr**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -28,7 +28,7 @@
 
 @media (prefers-reduced-motion: no-preference) {
   a:nth-of-type(2) .logo {
-    animation: logo-spin infinite 20s linear;
+    animation: logo-spin infinite 1s linear;
```
Comment (josephse91) [inline src/App.css]: This is too fast for the user. Lets stick to 5s or more.

File: src/App.jsx [hunk 2]
```diff
@@ -15,18 +15,19 @@ function App() {
...
           count is {count}
         </button>
         <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
+          Edit <code>src/App.jsx</code> and save to testtt HMR
         </p>
       </div>
       <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
+        Click on the Vite and React logos to learn mori
```
Comment (josephse91) [inline src/App.jsx]: there is a typo with test on line 24 and a typo with "more" on line 28

**Review by josephse91 — CHANGES_REQUESTED** [review]

File: src/App.css (hunk 1 above)
Comment (vcjr) [reply src/App.css]: Changes made.

File: src/App.jsx (hunk 2 above)
Comment (vcjr) [reply src/App.jsx]: Made sure to fix the typo here. Thank you for, great catch. It would've been super embarrassing!!!!

**Review by josephse91 — APPROVED** [review]

**Pull request merged**